pkgs/sdk/server/src/Internal/FDv2DataSources/FDv2DataSource.cs

```diff
@@ -0,0 +1,259 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using LaunchDarkly.Sdk.Server.Interfaces;
+using LaunchDarkly.Sdk.Server.Subsystems;
+
+using static LaunchDarkly.Sdk.Server.Subsystems.DataStoreTypes;
+
+namespace LaunchDarkly.Sdk.Server.Internal.DataSources
+{
+    internal static class FDv2DataSource
+    {
+        /// <summary>
+        /// Creates a new FDv2 data source.
+        /// </summary>
+        /// <param name="updatesSink">the sink that receives updates from the active source</param>
+        /// <param name="initializers">List of data source factories used for initialization</param>
+        /// <param name="synchronizers">List of data source factories used for synchronization</param>
+        /// <param name="fdv1Synchronizers">List of data source factories used for FDv1 synchronization</param>
```

Comment:
In configuration I have been doing this as a single item, but do we want to support a list?